klippy/serialhdl.py

```diff
@@ -5,16 +5,17 @@
 # This file may be distributed under the terms of the GNU GPLv3 license.
 import logging, threading, os
 import serial
-
+import errno
```

Comment:
this dhould probably be moved to line 6 where all stdlib imports are defined

Reply:
@thijstriemstra ,

Thank you. I combined the `errno` import with all the other stdlib imports.